src/Twig/Extension/DateTimeExtension.php

```diff
@@ -57,14 +72,20 @@ public function getFilters()
      */
     public function formatDate($date, $pattern = null, $locale = null, $timezone = null, $dateType = null)
     {
-        if (null !== $pattern) {
-            return $this->helper->format($date, $pattern, $locale, $timezone);
-        }
+        @trigger_error(sprintf(
+            'The %s method is deprecated since 2.x and will be removed on 3.0. '.
+            'Use %s::%s instead.',
+            __METHOD__,
+            DateTimeRuntime::class,
+            __METHOD__,
+        ), \E_USER_DEPRECATED);
```

Comment:
IMO these messages should encourage the user to use the new `sonata_*` filters instead of runtime class.

Reply:
Done